flax/linen/module.py

```diff
@@ -96,10 +127,16 @@ def _module_repr(module: 'Module', num_spaces: int = 4):
   cls = type(module)
   cls_name = cls.__name__
   rep = ''
-  attributes = {k: v for k, v in cls.__annotations__.items()
-                if k not in ('parent', 'name')}
-  child_modules = {k: v for k, v in module._state.children.items()  # pytype: disable=attribute-error
-                   if isinstance(v, Module)}
+  attributes = {
```

Comment:
This also seems worse, I think the initial version is easier to read.

Reply:
yeah, but black (and thus pyink) are mostly non-configurable